Modules/Microsoft365DSC/Modules/M365DSCReport.psm1

```diff
@@ -427,7 +427,10 @@ Local path of the destination configuraton.
 Array that contains the list of configuration components for the source.
 
 .Parameter DestinationObject
-Array that contains the list of configuration components for the destination. |
+Array that contains the list of configuration components for the destination.
+
+.Parameter ExcludedProperties
+Array that contains the list of configuration components for the destination.
```

Comment:
Please check the parameter’s description. Looks like it’s the one of destinationobject.